include/picongpu/plugins/PhaseSpace/PhaseSpace.hpp

```diff
@@ -41,6 +43,7 @@
 
 #include <string>
 #include <utility>
+#include <mpi.h>
```

Comment:
If I remember correctly, MPI is quite sensitive to be included *before* standard libs...?
(I might also be too old already and this is a flash back memory from the time I was still young and naïve)

Reply:
No there is no need to place the MPI include to the top. Even if the include is the first, includes from other files will be on top of the MPI include.

Reply:
true.